ovs/client.go

```diff
@@ -317,6 +317,14 @@ func SetSSLParam(pkey string, cert string, cacert string) OptionFunc {
 	}
 }
 
+// SetTCPParam configures the OVSDB connection using a TCP format ip:port
+// for use with all ovs commands
+func SetTCPParam(ip string, port string) OptionFunc {
```

Comment:
Combine these into a single parameter: `addr` that accepts the IP and string together.